source/help/settings/account-settings.md

```diff
@@ -52,23 +52,34 @@ Sessions are created when you log in with your email and password to a new brows
 ## Notifications
 Settings to configure desktop notifications, desktop notification sounds, email notifications, and words that trigger mentions.
 
-#### Send Desktop Notifications
-Desktop notifications appear at the bottom right corner of your main monitor. The desktop notification preference you choose in *Account Settings* applies globally, but this preference is customizable for each channel from the channel name drop-down menu. Desktop notifications are available on Firefox, Safari, and Chrome.
+#### Desktop Notifications
+Desktop notifications appear at the bottom right corner of your main monitor when there is activity in Mattermost. 
 
-#### Desktop Notification Sounds
-A notification sound plays for all Mattermost posts that would fire a desktop notification, unless *Desktop Notification Sound* is turned off. Desktop notification sounds are available on Firefox, Safari, Chrome, Internet Explorer, and Edge.
+##### Send Desktop Notifications
+
+Choose what activity triggers a desktop notification. This setting applies globally, but this preference is customizable for each channel from the channel name drop-down menu. Desktop notifications are available on Firefox, Safari, and Chrome.
+
+##### Notification Sound
+
+Notification sounds fire for any activity that would trigger a desktop notification. Notification sounds are available on IE11, Edge, Safari and Chrome.
+
+##### Notification Duration
+
+Sets how long desktop notifications will remain on screen. If set to **Unlimited**, the user will have to manually clear each notification as it appears on screen.
```

Comment:
Are there certain browsers "Unlimited" isn't supported on that we should add here